tests/_support/AcceptanceTester.php

```diff
@@ -12,15 +12,423 @@
  * @method void am($role)
  * @method void lookForwardTo($achieveValue)
  * @method void comment($description)
- * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = NULL)
+ * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = null)
  *
  * @SuppressWarnings(PHPMD)
-*/
+ */
 class AcceptanceTester extends \Codeception\Actor
 {
-    use _generated\AcceptanceTesterActions;
+	use _generated\AcceptanceTesterActions;
 
-   /**
-    * Define custom actions here
-    */
+	/**
+	 * Install joomla CMS
+	 *
+	 * @Given Joomla CMS is installed
+	 */
+	public function joomlaCMSIsInstalled()
+	{
+		// throw new \Codeception\Exception\Incomplete("Step `Joomla CMS is installed` is not defined");
+	}
+
+	/**
+	 * @When Login into Joomla administrator with username :arg1 and password :arg1
+	 */
+	public function loginIntoJoomlaAdministrator($username, $password)
+	{
+		$I = $this;
+		$I->doAdministratorLogin($username, $password);
+	}
+
+	/**
+	 * @Then I see administrator dashboard
+	 */
+	public function iSeeAdministratorDashboard()
+	{
+		$I = $this;
+		$I->waitForPageTitle('Control Panel', 4);
+	}
+
+	/**
+	 * @Given There is a Add Content link
+	 */
+	public function thereIsAAddContentLink()
+	{
+		$I = $this;
+		$I->amOnPage('administrator/index.php?option=com_content&view=articles');
+		$I->clickToolbarButton('New');
+	}
+
+	/**
+	 * @When I add the content with title :arg1
+	 */
+	public function iAddTheContentWithTitle($title)
+	{
+		$I = $this;
+		$I->fillField(['id' => 'jform_title'], $title);
+	}
+
+	/**
+	 * @When I add content body :arg1
+	 */
+	public function iAddContentBody($body)
+	{
+		$I = $this;
+		$I->click('Toggle editor');
+		$I->fillField(['id' => 'jform_articletext'], $body);
+	}
+
+	/**
+	 * @When I save an article
+	 */
+	public function iSaveAnArticle()
+	{
+		$I = $this;
+		$I->clickToolbarButton('Save');
+	}
+
+	/**
+	 * @Then I should see the :arg1 message
+	 */
+	public function iShouldSeeTheMessage($mesage)
```

Comment:
Spelling of variable should be correct, it should be $message, change it in the function also